zephyr/CMakeLists.txt

```diff
@@ -458,6 +458,11 @@ zephyr_library_sources_ifdef(CONFIG_LOG_BACKEND_SOF_PROBE
 
 # Optional SOF sources - depends on Kconfig - WIP
 
+# Optional math utility
+zephyr_library_sources_ifdef(CONFIG_MATH_LUT_TRIG_FIXED
+	${SOF_MATH_PATH}/lut_trig.c
+)
+
```

Comment:
Please move this next to the other SOF_MATH_PATH (#8620)

Reply:
Wrong PR? There's no change to CMakeLists.txt in that.